frontend/institution/create_inst_form.html

```diff
@@ -268,17 +268,39 @@
                                         <div ng-message="required">Este campo é obrigatório!</div>
                                     </div>
                                     </md-input-container>
+                                    <md-input-container ng-if="configInstCtrl.isSubmission" class="md-block" flex-gt-sm flex-sm>
+                                        <label>Seu nome</label>
+                                        <input name="admin" required ng-model="configInstCtrl.newInstitution.admin.name">
+                                    </md-input-container>
                                 </form>
                             </div>
                             <load-circle layout="row" layout-align="center center" ng-if="configInstCtrl.loadingSaveInstitution"></load-circle>
                         </md-card-content>
                         <md-card-actions layout="row" layout-align="end center">
-                            <md-button class="md-raised" ng-if="!configInstCtrl.loadingSaveInstitution" md-colors="{background: 'teal-500'}" ng-click="configInstCtrl.submit($event)">
+                            <md-button class="md-raised" ng-if="!configInstCtrl.loadingSaveInstitution"
+                            md-colors="{background: 'teal-500'}" ng-click="configInstCtrl.submit($event)">
                                 FINALIZAR
                             </md-button>
                         </md-card-actions>
                     </div>
                 </md-card>
+                <md-card ng-if="configInstCtrl.getStep(4)" layout-margin>
+                    <div flex>
+                        <md-card-content layout-margin>
+                            <div layout="column">
+                                <div style="font-weight: 500; text-align: center;">
+                                    <p>Sua solicitação de convite foi enviada e esta em análise pelo administrador
+                                    de sua instituição na Plataforma CIS.</p>
```

Comment:
The sender is the admin, change to "foi enviada e está em análise pelo administração da Plataforma Virtual CIS"